### DISCUSSION TITLE
docs/guide/eip-snat/

### DISCUSSION THREAD

**Discussion by giscus** [kubeovn/kube-ovn#1642] [Announcements]
# docs/guide/eip-snat/

https://kubeovn.github.io/docs/guide/eip-snat/

**Comment by iceCreeam** [comment]
该功能是否可以指定VPC？

**Reply by oilbeater** [reply]
目前只能在默认 VPC 使用，用户自定义 VPC 要参考 [VPC 网关](https://kubeovn.github.io/docs/guide/vpc/#vpc_2)
